packages/mattermost-redux/src/reducers/entities/posts.ts

```diff
@@ -271,45 +271,59 @@ export function handlePosts(state: RelationOneToOne<Post, Post> = {}, action: Ge
     }
 }
 
-function handlePostReceived(nextState: any, post: Post) {
-    if (!shouldUpdatePost(post, nextState[post.id])) {
-        return nextState;
+function handlePostReceived(nextState: any, post: Post, nestedPermalinkLevel?: number) {
+    let currentState = nextState;
```

Comment:
Why not just call the function parameter `currentState`

Reply:
The linter doesn't like when we reassign function parameters (occurs a few lines below this), so we need to declare it within the function.